diff --git a/node_modules/wechatpay-node-v3/src/index.js b/node_modules/wechatpay-node-v3/src/index.js
index 13d1007..7c9360c 100644
--- a/node_modules/wechatpay-node-v3/src/index.js
+++ b/node_modules/wechatpay-node-v3/src/index.js
@@ -1,5 +1,5 @@
 const { Pem, Aes, Rsa } = require('./lib');
-const { Form, Notification } = require('./middleware');
+// const { Form, Notification } = require('./middleware');
 const {
   REQUEST_METHOD,
   REQUEST_WITH,
@@ -79,7 +79,7 @@ class Wechatpay {
     this.middleware = this.middleware.bind(this);
   }
 
-  static Form = Form;
+  // static Form = Form;
 
   /**
    * Cast the `number` to `int`
@@ -525,7 +525,7 @@ class Wechatpay {
    * @returns {Notification} - The `notification` middleware
    */
   middleware(type = 'notification') {
-    const map = { notification: Notification };
+    const map = { };
     const name = Object.keys(map).find((key) => key === type.toLowerCase());
 
     return new map[name](this);
diff --git a/node_modules/wechatpay-node-v3/src/middleware/index.js b/node_modules/wechatpay-node-v3/src/middleware/index.js
index 995b0c9..d7b693e 100644
--- a/node_modules/wechatpay-node-v3/src/middleware/index.js
+++ b/node_modules/wechatpay-node-v3/src/middleware/index.js
@@ -1,5 +1,5 @@
-const Form = require('./form');
-const Notification = require('./notification');
+// const Form = require('./form');
+// const Notification = require('./notification');
 
-module.exports = { Form, Notification };
+module.exports = { };
diff --git a/node_modules/wechatpay-node-v3/src/middleware/notification.js b/node_modules/wechatpay-node-v3/src/middleware/notification.js
index 585c575..0e86b02 100644
--- a/node_modules/wechatpay-node-v3/src/middleware/notification.js
+++ b/node_modules/wechatpay-node-v3/src/middleware/notification.js
@@ -1,4 +1,4 @@
-const { formidable } = require('formidable');
+// const { formidable } = require('formidable');
 
 const {
   PLATFORM_SERIAL_NO,
@@ -21,7 +21,7 @@ class Notification {
 
   /**
    * @property {object} opts - The `formidable` options
-   * @see https://github.com/node-formidable/formidable#options
+   * @see https://github.com/node-formidable/formidable#options - link for options
    */
   opts = {}
 
@@ -35,7 +35,7 @@ class Notification {
    * @returns {function} - The koa middleware
    */
   koa() {
-    const form = formidable(this.opts);
+    // const form = formidable(this.opts);
 
     const promise = (req) => new Promise((resolve, reject) => {
       form.parse(req, (err, fields, files) => {
@@ -91,7 +91,7 @@ class Notification {
    * @returns {function} - The express middleware
    */
   express() {
-    const form = formidable(this.opts);
+    // const form = formidable(this.opts);
 
     const promise = (req) => new Promise((resolve, reject) => {
       form.parse(req, (err, fields, files) => {
@@ -148,7 +148,7 @@ class Notification {
    * @returns {function} - The express middleware
    */
   egg() {
-    const form = formidable(this.opts);
+    // const form = formidable(this.opts);
 
     const promise = (ctx) => new Promise((resolve, reject) => {
       form.parse(ctx.req, (err, fields, files) => {
