diff --git a/node_modules/wechatpay-node-v3/src/index.js b/node_modules/wechatpay-node-v3/src/index.js
index 9276d49..f9f0951 100644
--- a/node_modules/wechatpay-node-v3/src/index.js
+++ b/node_modules/wechatpay-node-v3/src/index.js
@@ -1,5 +1,5 @@
 const Pem = require('./pem');
-const Formidable = require('formidable');
+// const Formidable = require('formidable');
 const { createHash } = require('crypto');
 const { readFileSync, writeFileSync } = require('fs');
 const { Aes, Rsa, request, Download, `SignType` } = require('./utils');
@@ -194,7 +194,7 @@
    * @property {string} nonce_str - The nonce_str from wechat.
    * @returns {object} - The payment detail.
    */
-  async notification(request) {
+  /*async notification(request) {
     const {
       'wechatpay-timestamp': timestamp,
       'wechatpay-nonce': nonce,
@@ -239,7 +239,7 @@
         reject(err);
       });
     });
-  }
+  }*/
 }
 
 WechatPay.SignType = SignType;
